groups/templates/groups/emails/new_comment.txt

```diff
@@ -6,4 +6,4 @@ A new comment has been posted to the discussion "{{ name }}" by {{ creator }}{%
 
 {% else %}.{% endif %}
 
-{% blocktrans with url=comment.get_absolute_url %}Reply to this email to post a response, or visit this URL to reply online: {{ site.domain }}{{ url }}{% endblocktrans %}
+{% blocktrans with url=comment.get_absolute_url domain=site.domain %}Reply to this email to post a response, or visit this URL to reply online: http://{{ domain }}{{ url }}{% endblocktrans %}
```

Comment:
You can get it [from the request](https://docs.djangoproject.com/en/1.8/ref/request-response/#django.http.HttpRequest.is_secure).